### ISSUE TITLE
When is the Deadline for Shipping Compounds to Wendy/Wilson?

### ISSUE THREAD

**Issue by mattodd** [OpenSourceMycetoma/Series-1-Fenarimols#32]
Hi all,

We spoke at a recent meeting (#31 #28) about the deadline for submitting compounds to @wwjvdsande and @wilson-lm for biological evaluation. Can someone please remind me what the **last date is (roughly) for receipt of compounds in Rotterdam**? This is relevant to @fantasy121 @dmitrij176 and @bendndi.

And is this deadline relevant to both _in vitro_ and _in vivo_ evaluation, or just _in vivo_?

**Comment by fantasy121** [comment]
The preferrable deadline of 15th July has lapsed. My understanding from our meeting is that this deadline is technically for the _in vitro_  but we need to have this result to narrow down the _in vivo_ candidates to save time/resource running _in vivo_. I think The hard deadline is from now till two more weeks (Mid-August) if we are to have enough time to run _in vivo_ this season, though this is really pushing it. I could be wrong. I think @Wilson-Lm can confirm?

tl-dr: it's ASAP from now.

Unfortunately I won't be able to submit new compounds for _in vivo_ testing this season due to postponing of lab work as my neighbourhood was the origin of a second wave of COVID in my state (It's the Crossroads Hotel cluster in NSW if you heard it on the news). So I decided to reschedule my lab return 2 weeks from when this cluster was discovered for safety reasons.

**Comment by Wilson-Lm** [comment]
Hi @fantasy121 @mattodd 
The absolute latest will be the end of the first week of august. Even that is already pushing it a lot. It also depends on how many compounds that will be sent. 

What we are currently worried about is the time left in the year 2020 to run the in vivo experiments. based experience, in vivo larvae work can only be reliably performed until end of September / early October. 

We'll need at least 2.5 weeks to perform the in vitro work and 3 to 4 weeks to perform the in vivo work. (this is of course if everything goes well). **So, if a reasonable amount of compounds arrive before the end of the first week of August, i believe that they can still be pushed through into the in vivo stage.** 

We are currently having a set back in our in vivo work. We were facing unreliable supplies for a while and that required us to change to a new one and the subsequent need to re-optimise the protocol to suit these batch of larvae. Manpower is also a problem at the moment due to safety reasons, but hopefully that will be solved in the next few weeks. By then we will be able to push through and hopefully get things back to where they are suppose to be.

**Comment by MFernflower** [comment]
@fantasy121 Are you still in QT?

**Comment by mattodd** [comment]
Hi @bendndi just pinging you to be sure you're aware re Ghana samples.
Thanks for clarification @Wilson-Lm - so just to follow-up: can we continue to send samples for in vitro analysis during the colder months?

**Comment by dmitrij176** [comment]
@Wilson-Lm I won't be able to make and send you additional analogs as was planned initially. In the current circumstances, theres no way that can be acomplished by the end of next week since I am not back to London yet. As soon as the lab work resumes, we will start working on another batch of molecules for the next screening season.

**Comment by fantasy121** [comment]
> @fantasy121 Are you still in QT?

I’m starting to resume lab work this week. Planning to do half-week lab work schedule for the time being to accommodate our faculty’s physical distancing requirements should other researchers in my group need to access to the lab also.

I’m ditto with Dmitrij about getting molecules for the next screening season, but like Mat has said, I’m happy to send molecules in the colder months as well if in vitro is possible in this period.

**Comment by Wilson-Lm** [comment]
@mattodd, yes of course! We can do the _in vitro_ analysis anytime irregardless of the season.  

@dmitrij176 and @fantasy121, sure, no worries with the 'delay'. As it is, we've already got a bit too much on our plates at the moment, so we would actually prefer to leave them for the next screening batch. You both are much welcomed to send them over when it's ready.

and just to clarify, the data for these future molecules will not be included in the current '_Fenarimol 2020_' manuscript. It will be used in another article.

**Comment by MFernflower** [comment]
@mattodd @fantasy121 That should be okay right? We do assume that we would probably get a few papers out of S1 before eventually trying something else?

**Comment by MFernflower** [comment]
@wwjvdsande @Wilson-Lm Any new in vitro results to share regarding Hung's compounds?

**Comment by Wilson-Lm** [comment]
@MFernflower From our initial testing with 1 _Madurella mycetomatis_ isolate, the following compounds showed potency under 25uM.
1. HPD14-1
2. HPD033-2P4
3. HPD033-3P3
4. HPD037-1

We are currently testing these 4 compounds on a panel of 10 _Madurella mycetomatis_ isolates. We're expecting to get the results in 2 - 3 weeks. We're slightly delayed due to the school and summer holidays. Now that I've finally returned to the Netherlands, I will be able to speed things up a notch.

**Comment by MFernflower** [comment]
Interesting to see that the alkyl fenarimols seem good @fantasy121

**Comment by MFernflower** [comment]
@fantasy121 @Wilson-Lm @mattodd 

Hopefully I got this right:

![August2020_WENDYANDWILSON](https://user-images.githubusercontent.com/3164942/91237483-d525cb80-e708-11ea-8511-82dc5b2e7f4d.png)

![compoundsv2](https://user-images.githubusercontent.com/3164942/91237504-dbb44300-e708-11ea-9b2a-241a1ffa6dce.png)

**Comment by fantasy121** [comment]
> @fantasy121 @Wilson-Lm @mattodd
> 
> Hopefully I got this right:
> 
> ![August2020_WENDYANDWILSON](https://user-images.githubusercontent.com/3164942/91237483-d525cb80-e708-11ea-8511-82dc5b2e7f4d.png)
> 
> ![compoundsv2](https://user-images.githubusercontent.com/3164942/91237504-dbb44300-e708-11ea-9b2a-241a1ffa6dce.png)

**Comment by fantasy121** [comment]
@MFernflower For compounds to come you can refer to #29 https://github.com/OpenSourceMycetoma/Series-1-Fenarimols/issues/29#issuecomment-630538844. I'm currently trying to make the ones in yellow.

**Comment by fantasy121** [comment]
> @mattodd @fantasy121 That should be okay right? We do assume that we would probably get a few papers out of S1 before eventually trying something else?

That's totally fine. As we discussed in our meetings, we should just include what we can in the 2020 paper and leave future data for the next paper.

**Comment by fantasy121** [comment]
@Wilson-Lm @wwjvdsande Can I also clarify the deadline for sending samples for in vitro screenings? I understand that in vitro can be done year round but if a particular in vitro result is to be included in the 2020 paper then what is the last date possible to send those compounds over? Or if any more compounds are sent over now, their in vitro results will be reserved for a future paper.

**Comment by Wilson-Lm** [comment]
Hey Hung, @fantasy121 , 
I missed this message amidst all the others from github (sorry!) 
In Vitro can be done all year yes and you're more than welcome to send over compounds anytime. For the 2020 paper, we can no longer do any testings. Technically we are able to to test the compounds in vitro, but we cannot do in vivo tests anymore as our in vivo larvae season has ended for this year. 
So, any compounds we now receive will be reserved for a future paper.
